PROJECT_flexbe_behaviors/src/PROJECT_flexbe_behaviors/manifests/example_behavior_manifest.py

```diff
@@ -0,0 +1,33 @@
+example_behavior = {
+	'name': 'Example Behavior',
+	'executable': {
+		'package_path': 'PROJECT_flexbe_behaviors.example_behavior_sm',
+		'class': 'ExampleBehaviorSM'
+	},
+
+	'author': 'Philipp Schillinger',
+	'date': 'Fri Aug 21 2015',
+	'tagstring': 'demo',
+	'description': """
+		This is a simple example for a behavior.
+	""",
+
+	# Contained behaviors
```

Comment:
Contained behaviors are given by just their *name* entry, no further information has been stored before. So this entry can be just a list of strings.

For example, if the *Example Behavior* would be contained somewhere, the manifest would say:
```python
    'contained_behaviors' [
        'Example Behavior',
    ]
```